mono/mini/interp/interp.c

```diff
@@ -2481,7 +2483,9 @@ interp_entry_general (gpointer this_arg, gpointer res, gpointer *args, gpointer
 	interp_entry (&data);
 }
 
-#ifdef MONO_ARCH_HAVE_INTERP_ENTRY_TRAMPOLINE
+#define interp_entry_from_trampoline NULL
```

Comment:
This isn't ideal, in that, it "works" even if there is a function. It'll just replace it, if it is later.
Maybe better to provide a function with unreachable?